scripts/ci/add_supported_release.py

```diff
@@ -71,8 +76,17 @@ def main() -> int:
     parser.add_argument(
         "--image-name", help="image to add a new supported version", type=str
     )
+    parser.add_argument(
```

Comment:
Does this align with the Enterprise `pipeline.py`?

Reply:
These two unaligned if you compare them. It's not worth the cost - we'll soon get rid of adding versions to the external mongodb database and switch to simple file in repo. But this is the next step I need to convince the team on :)